docs/source/markdown/podman-pod-create.1.md

```diff
@@ -120,6 +120,15 @@ Add a DNS alias for the container. When the container is joined to a CNI network
 
 Disable creation of /etc/hosts for the pod.
 
+#### **--pid**=*pid*
+
+Set the PID mode for the pod Default is to create a private PID namespace for the pod
+
+    host: use the host’s PID namespace for the pod
+    ns: join the specified PID namespace
+    private: create a new namespace for the pod (default)
+    pod: The pod's PID namespace will be used for all containers
```

Comment:
I would just omit this... We should still accept it, but it doesn't need to but documented